benchmarks/aes/Dockerfile

```diff
@@ -0,0 +1,92 @@
+# MIT License
+#
+# Copyright (c) 2022 David Schall and EASE lab
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+#---------- PYTHON -----------#
+# First stage (Builder):
+# Install gRPC and all other dependencies
+FROM vhiveease/python-slim:latest as aesPythonBuilder
+WORKDIR /py
+COPY ./benchmarks/aes/python/requirements.txt ./
+RUN pip3 install --user -r requirements.txt
+COPY ./utils/tracing/python/tracing.py ./
+COPY ./benchmarks/aes/python/server.py ./
+COPY ./benchmarks/aes/proto/helloworld_pb2_grpc.py ./
+COPY ./benchmarks/aes/proto/helloworld_pb2.py ./
+
+# Second stage (Runner):
+FROM vhiveease/python-slim:latest as aesPython
+COPY --from=aesPythonBuilder /root/.local /root/.local
+COPY --from=aesPythonBuilder /py /app
+WORKDIR /app
+# ENV PATH=/root/.local/bin:$PATH
+ENTRYPOINT [ "python", "/app/server.py" ]
+
+
+
+
+#---------- GoLang -----------#
+# First stage (Builder):
+FROM golang:1.16-alpine AS aesGoBuilder
```

Comment:
this should be like for Python (line 26): create a base vHive image

Reply:
What is the reason?
I mean its not a big thing to change it but I want to understand the benefit. It just adds one more layer of dependency.
Is it because of the tracing module?
Or is it than you can just update the base image and automatically all functions will be upgraded?
I just want to understand

Reply:
Ok thanks. Then its the same I was thinking. Nice 👍🏽

Reply:
Please check if the one I have used right now is fine.